src/odemis/dataio/tiff.py

```diff
@@ -2358,10 +2383,14 @@ def _getOMEXML(self, tfile):
         # It's OME TIFF, if it has a valid ome-tiff XML in the first T.TIFFTAG_IMAGEDESCRIPTION
         tfile.SetDirectory(0)
         desc = tfile.GetField(T.TIFFTAG_IMAGEDESCRIPTION)
-
-        if (desc and ((desc.startswith(b"<?xml") and b"<ome " in desc.lower()) or
-                      desc[:4].lower() == b'<ome')):
+        if (desc and ((b"<?xml" and b"<ome " in desc.lower() and not desc.lower().startswith(b"<ome ")) or
+                desc[:4].lower() == b'<ome')):
             try:
+                pattern = b'<?xml'
+                start_index = desc.find(pattern)
```

Comment:
Did you try using a regex? I think it'd be more powerful and cleaner to just use a regex, to search for it, and then use the `if` if there is a match. This avoids parsing the description string 4 times.
Also, it should work if it starts directly with `<ome`. Please, also only match if it's at the beginning of a line.